Issue #62, Issue #71, Issue #89, cleanup.

alegag opened the pull request:
Issue #89: Added methods show() and hide() in addition to toggle().

Issue #71: Added an optional autoClose=true to close the dialog on date/time selection.

Issue #62: Option for 24-hour in an inner-circle.

Removed "picker" variable definied and not used.

Removed "start" variable definied and not used.

Removed "spaces" mixed with "tabs".

alegag commented:
@puranjayjain Sorry, I saw the guideline after the pull request.  I wanted to make multiple commit at first, should have !  Next time !

alegag commented on src/js/mdDateTimePicker.js:
```diff
@@ -42,6 +43,7 @@ class mdDateTimePicker {
 		this._ok = ok
 		this._cancel = cancel
 		this._colon = colon
+		this._auto_close = auto_close
```
Fixed camelCase is last commit.

alegag commented on dist/js/mdDateTimePicker.js:
```diff
@@ -97,6 +100,7 @@
 			this._ok = ok;
 			this._cancel = cancel;
 			this._colon = colon;
+			this._auto_close = auto_close;
```
Fixed camelCase is last commit.

alegag commented on dist/js/mdDateTimePicker.js:
```diff
@@ -83,7 +84,9 @@
-			    colon = _ref$colon === undefined ? !0 : _ref$colon;
+			    colon = _ref$colon === undefined ? !0 : _ref$colon,
+			    _ref$auto_close = _ref.auto_close,
+			    auto_close = _ref$auto_close === undefined ? false : _ref$auto_close;
```
Fixed camelCase is last commit.

puranjayjain commented:
@alegag good work you can do multiple commits still I'll keep this open until you do no problem. You can do the multiple commits and in the end squash them together in one. I'll help you if you get stuck don't worry.

alegag committed:
Cleanup.

Removed unused variables.

Removed spaces where there were tabs in the mix.

alegag committed:
Added methods show() and hide() in addition to toggle().

alegag committed:
Added an optional autoClose=true to close the dialog on date/time selection.

alegag commented:
@puranjayjain Here you go ! As three different commits for the two issues and the little cleanup.

alegag committed:
autoClose feature and minute-draggability incompatibility.

When "autoClose=true", the minute-draggability cause the minutes to be
marked as "undefined" when the "currently selected minute" don't match
one of the minute on the dial.  Also, as we need to click the time to
close the dial, clicking the already-selected minute don't do anything
(as this is the drag-fake-needle).  To avoid that, disable the
draggability feature on minute dial when autoClose=true.

alegag committed:
Option for 24-hour in an inner-circle.

alegag commented:
@puranjayjain Okay, last commit I was pulling for now.  That commit cover issue #62 as an optional feature (needs to set inner24=true to use it).

That commit touch the SCSS.  I did not regenerated all the themes as I'm not familiar with the procedure.

I'll let your review all of this now.

puranjayjain commented:
ok @alegag I'll do it

puranjayjain commented:
Nice work @alegag thank you merging it now.
